File: models/crear_prestamo.go
package models

type Prestamo struct {
	Cedula            string  `json:"cedula"`
	Monto             float64 `json:"monto"`
	Interes           int64   `json:"interes"`
	Plazo             int64   `json:"plazo"`
	Numero_prestamo   string  `json:"numero_prestamo"`
	Fecha_prestamo    string  `json:"fecha_prestamo"`
	Primera_cuota     string  `json:"primera_cuota"`
	Total_pagar       string  `json:"total_pagar"`
	Tipo              string  `json:"tipo"`
	Desembolso        float64 `json:"desembolso"`
	Prestamo_anterior string  `json:"prestamo_anterior"`
	Fecha_final       string  `json:"fecha_final"`
}

File: js/crearprestamo.js
function crearprestamo(){
let cedula = document.getElementById("cedula").value;
let tipo_prestamo = document.getElementById("tipo_prestamo").value;
let monto = document.getElementById("monto").value;
let interes = document.getElementById("interes").value;
let plazo = document.getElementById("plazo").value;
let numero_prestamo = document.getElementById("numero_prestamo").value;
let fecha_prestamo = document.getElementById("fecha_prestamo").value;
let primera_cuota = document.getElementById("primera_cuota").value;
let desembolso = document.getElementById("desembolso").value;
let prestamo_anterior = document.getElementById("prestamo_anterior").value;
let total_pagar = document.getElementById("total_pagar").value;
let fecha_final = "2022-10-01";
var settings = {
  "async": true,
  "crossDomain": true,
  "url": "http://127.0.0.1:8081/prestamos",
  "method": "POST",
  "headers": {
    "content-type": "application/json",
    "cache-control": "no-cache",
    "postman-token": "0f85aab2-241e-da90-08bc-5f007b9c4d9d"
  },
  "processData": false,
  "data": "{\n\"cedula\": \""+cedula+"\",\n\"monto\": "+monto+",\n\"interes\": "+interes+",\n\"plazo\": "+plazo+",\n\"numero_prestamo\": \""+numero_prestamo+"\",\n\"fecha_prestamo\": \""+fecha_prestamo+"\",\n\"primera_cuota\": \""+primera_cuota+"\",\n\"total_pagar\": \""+total_pagar+"\",\n\"tipo\": \""+tipo_prestamo+"\",\n\"desembolso\": "+desembolso+",\n\"prestamo_anterior\": \""+prestamo_anterior+"\",\n\"fecha_final\": \""+fecha_final+"\"\n}"
}

$.ajax(settings).done(function (response) {
  console.log(response);
});
}